src/toGamut.js

```diff
@@ -118,8 +118,8 @@ toGamut.returns = "color";
 // `Oklch` space. These are created in the `Oklab` space, as it is used by the
 // DeltaEOK calculation, so it is guaranteed to be imported.
 const COLORS = {
-	WHITE: { space: "oklab", coords: [1, 0, 0] },
-	BLACK: { space: "oklab", coords: [0, 0, 0] }
+	WHITE: { space: oklab, coords: [1, 0, 0] },
```

Comment:
Just to make sure this isn't forgotten, a resolution of how color.js wishes to ensure a good white/black should be decided as the current Oklab matrix is not exact enough to give a decent white and will introduce a notable error.

Whether each space reports the optimal white/black, a precalculated perfect white in Oklab is used, a generic method using XYZ D65 white and chromatically adapting the value to the target color space's white point is used, a recalculating Oklab with 64 bit precision eliminate the inaccuracies near achromatic values is used, or some other method is used, a solution should be considered to reduce errors here.

Reply:
It should be noted that I do have a pull request to fix the issues related to chromatic adaptation here: https://github.com/LeaVerou/color.js/pull/354. I imagine this could be rebased on that to test for accuracy.

Reply:
That PR is now merged, so the CA is now good. I also put the same values in the CSS Color 4 sample code.

Reply:
I've opened another PR with new Oklab matrices for working in 64 bit. https://github.com/LeaVerou/color.js/pull/357

Reply:
Due to my own carelessness, I had updated the inverse D50 <-> D65 matrix incorrectly. The last fix is here: https://github.com/LeaVerou/color.js/pull/360.

Once #357 and #360 are approved, all the fixes for proper GMA should be in place and then this issue can be resolved.

Reply:
Since #357 and #360 are now on main, it no longer matters that `oklab(1 0 0)` is used for white as it should be plenty accurate now.

Reply:
Great, I've merged main in, so we can test both the toGamut logic and white conversions [here](https://deploy-preview-352--colorjs.netlify.app/).